summarization/extractor_output_preprocessor.py

```diff
@@ -146,7 +171,8 @@ def _get_tokenized_summary_sentence_from_index(
         # retain and set alphanumeric characters
         return self._get_alphanumeric_tokens(word_tokenized_text)
 
-    def _get_sentence_object(self, summarized_text_index, normal_text_index):
+    def _get_sentence_object_for_summarized_sentence(
+            self, summarized_text_index, normal_text_index):
```

Comment:
Method name changed to reflect that it is specific to instantiating objects for normal ( summarized) sentence